shared/constants/security-provider.ts

```diff
@@ -17,7 +17,43 @@ export const SECURITY_PROVIDER_CONFIG: Readonly<SecurityProviderConfig> = {
     tKeyName: 'blockaid',
     url: 'https://blockaid.io/',
   },
-};
+} as const;
+
+/** The reason, also referred to as the attack type, provided in the PPOM Response  */
+export enum BlockaidReason {
+  /** Approval for a malicious spender  */
+  approvalFarming = 'approval_farming',
+  /** Malicious signature on Blur order  */
+  blurFarming = 'blur_farming',
+  /** A known malicous site invoked that transaction  */
+  maliciousDomain = 'malicious_domain',
+  /** Malicious signature on a Permit order  */
+  permitFarming = 'permit_farming',
+  /** Direct theft of native assets (ETH/MATIC/AVAX/ etc …)  */
+  rawNativeTokenTransfer = 'raw_native_token_transfer',
+  /** Malicious signature on a Seaport order  */
+  seaportFarming = 'seaport_farming',
+  /** setApprovalForAll for a malicious operator  */
+  setApprovalForAll = 'set_approval_for_all',
+  /** Malicious raw signature from the user   */
+  signatureFarming = 'signature_farming',
```

Comment:
is this supposed to be based on[ this comment](https://github.com/metamask/metamask-planning/issues/396#issuecomment-1642092127) from Barbara

```suggestion
  rawSignatureFarming = 'raw_signature_farming',
```